test/src/test/java/org/corfudb/integration/LogReplicationSinkWriterIT.java

```diff
@@ -139,13 +135,14 @@ private void prepareTablesToVerify(int start, int splitter, int end) throws Exce
             Table<StringKey, IntValueTag, Metadata> tableActive =
                     corfuStoreActive.openTable(NAMESPACE, mapName, StringKey.class,
                             IntValueTag.class, Metadata.class,
-                            TableOptions.fromProtoSchema(IntValueTag.class, TableOptions.builder().build()));
+                            TableOptions.fromProtoSchema(IntValueTag.class));
             mapNameToMapActive.put(mapName, tableActive);
 
+            // Open tables on Sink side with different schema options
             Table<StringKey, IntValueTag, Metadata> tableStandby =
                     corfuStoreStandby.openTable(NAMESPACE, mapName, StringKey.class,
                             IntValueTag.class, Metadata.class,
-                            TableOptions.fromProtoSchema(IntValue.class, TableOptions.builder().build()));
+                            TableOptions.fromProtoSchema(BackupRestoreTag.class));
```

Comment:
both IntValueTag and BackupRestoreTag have 'is_federated' true.  What are we testing then?  We are not testing that the difference in the flag value caused an entry to be dropped.

Reply:
it's using backup_restore_tag to verify the already existed entry in registry table is not overwritten. It was using is_federated flag to do that since we didn't rely on dynamic streams before